corehq/sql_accessors/migrations/0032_exclude_deleted_cases.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from corehq.sql_db.operations import HqRunSQL, RawSQLMigration
+
+migrator = RawSQLMigration(('corehq', 'sql_accessors', 'sql_templates'), {})
```

Comment:
it looks like this needs: `'RELATIONSHIP_TYPE_EXTENSION': CommCareCaseIndexSQL.EXTENSION` to fix build